constraintFunctions.f90

```diff
@@ -149,6 +149,9 @@ subroutine getEnvVarsAtT( t )
     character(len=maxEnvVarNameLength) :: this_env_var_name, orderedEnvVarNames(size( envVarNames ))
     logical :: pressure_set, rh_set, temp_set
 
+    ! TODO, is this the best place to call calcTheta?
```

Comment:
Btw, that large block of comment contains a lot of info. part of it is explanation of what the subroutine does and part are details. so I think it should be split up and some info moved to the header (where ??? is). Do you agree?